shortcodes.php

```diff
@@ -1464,6 +1460,34 @@ function sc_expert_photos($atts = Array())
 add_shortcode('expert_photos', 'sc_expert_photos');
 
 
+/**
+ * Feature post meta.
+ *
+ * @return string.
+ * @author Cadie Brown
+ **/
+function sc_feature_post_meta($atts = Array())
+{
+	global $post;
+
+	$css = (isset($atts['css'])) ? $atts['css'] : '';
+
+	$byline = get_post_meta($post->ID, 'author_byline', True);
+	$byline = ($byline != '') ? $byline : get_the_author();
+
+	ob_start()?>
+	<div class="<?php echo $css; ?>" id="meta">
+		<div>
+			<p id="byline-date">By <?php echo $byline; ?> <span class="hidden-mobile">|</span><br class="visible-mobile"> <?php echo date('F j, Y', strtotime($post->post_date)); ?></p>
+		</div>
+	</div>
+	<?
+	$html = ob_get_contents(); ob_end_clean();
```

Comment:
You can use `ob_get_clean();` here instead of `ob_get_contents(); ob_end_clean();` to simplify the process of closing and retrieving the output buffer's contents.